migtests/scripts/resumption.py

```diff
@@ -191,16 +194,22 @@ def run_command(command, allow_interruption=False, interrupt_after=None):
             print("\nCommand Output:\n")
             for line in stdout.splitlines():
                 print(line)
-
         if stderr:
             print("\nCommand Errors:\n")
             for line in stderr.splitlines():
                 print(line)
+            sys.exit(1)
 
-        completed = process.returncode == 0 and not interrupted
+        # If interrupted, check the exit code
+        if interrupted:
+            if process.returncode not in {1, -9, 137}:  # -9 and 137 are SIGKILL variations
```

Comment:
there are atleast three signal we should test specifically(Not user if there are more or any custom/USR signal can do the same)
- SIGTERM, SIGKILL and SIGINT 

Lets handle these and update the comment.

Reply:
In case of these signal we should also capture the error message specifically of the import data command and print it here.

Is that already taken care of for every intermediate runs?

Reply:
The stdout or stderr of the commands will already be printed. Then if we encounter such a situation, we will exit as well. 
Yes it will happen for intermediate runs.

Reply:
Voyager handles SIGTERM internally and returns exit code 1 in that case. We aren't using SIGINT anywhere in the script so if we receive its exit code we should flag it.
Currently we are running SIGTERM and SIGKILL. If we should add SIGINT as well, we can take it up in the upcoming enhancements.